lib/model/model.go

```diff
@@ -1920,11 +1930,6 @@ func (m *Model) internalScanFolderSubdirs(ctx context.Context, folder string, su
 			break
 		}
 
-		// We test each path by joining with "root". What we join with is
-		// not relevant, we just want the dotdot escape detection here. For
-		// historical reasons we may get paths that end in a slash. We
-		// remove that first to allow the rootedJoinedPath to pass.
-		sub = strings.TrimRight(sub, string(fs.PathSeparator))
```

Comment:
Why removing this?

Reply:
`routedJoinedPath` doesn't exist anymore and scanner/fs doesn't care about trailing path separators in the prefixes/subs (also I believe in the sub unification before calling `internalScanFolderSubdirs` we already format paths).